### ISSUE TITLE
BlenderBIM: IFC project appears as if not having been created upon reopening

### ISSUE THREAD

**Issue by cvillagrasa** [IfcOpenShell/IfcOpenShell#1323]
Hi! I might be missing some obvious required step, since I just started using BlenderBIM.

What I'm experiencing is that after saving and reopening Blender, the Scene and Object tabs appear as if the IFC project or any IFC class for the objects had not yet been setup, while the scene tree still looks OK.

![Selection_026](https://user-images.githubusercontent.com/57069798/108187512-d6134500-710e-11eb-9123-25cd6220c07a.png)

If I just try to click on Create Project again, an error occurs because it is already created:

![imagen](https://user-images.githubusercontent.com/57069798/108188645-1aebab80-7110-11eb-82ea-b06f19562d5a.png)

Tested on Windows 10 and Ubuntu 20.10, BlenderBIM version 0.0.210131

**Comment by Moult** [comment]
The latest version has a rather different paradigm shift, and the UX hasn't yet quite been polished to guide users to the right usage of the add-on just yet, so it's not your fault :)

Saving the Blender file only saves the ... well, the Blender file. The actual BIM model is not Blender, instead, it's stored in IFC. Therefore, you need to export your IFC file too. Then, Blender will be able to remember your BIM information.

**Comment by cvillagrasa** [comment]
Awesome! indeed, after exporting an IFC file once, everything works as expected.

Thanks Dion.

**Comment by Moult** [comment]
There is now a check when you save to make sure others don't run into this same issue.

**Comment by cvillagrasa** [comment]
Thank you! I had already closed the issue since I didn't consider it a real bug for alpha software.

I've tried to test the new check but somehow I am unable to git clone the repo and make it work. I get some problem either importing `ifcopenshell.util.type` or `file_open_status from ifcopenshell.ifcopenshell_wrapper`, depending on whether alongside blenderbim, I also update ifcopenshell-python files to the latest or not.

So... I'll just stick to version 210131 and wait for the next release. Keep up with this massive piece of software!
